src/ca-table.h

```diff
@@ -285,9 +287,15 @@ class Backend {
   virtual std::unique_ptr<TableBuilder> Create(const char* path,
                                                const TableOptions& options) = 0;
 
-  virtual std::unique_ptr<Table> Open(const char* path) = 0;
+  // NOTE: The fd ownership is transferred to the created object so it is
+  // responsible for closing the file when it is no longer needed.
+  virtual std::unique_ptr<Table> Open(const char* path, int fd,
+                                      const struct stat& st) = 0;
```

Comment:
I didn't want to pass kj::AutoCloseFd through the Backend interface as it is
1) defined in ca-table.h that defines a piblic API,
2) I view using the kj library as a kind of implementation detail.

I could pass  kj::AutoCloseFd for sure, but then I would suggest to move Backend from ca-table.h to some internal header (e.g table-backend.h). I think it is not very useful anyway.